CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 16.1.0
```

Comment:
We'll want this to be `17.0.0` as the change makes the config stricter. We use the minor when when we make a change that makes the config more permissive.

Reply:
OK. Fixed via c259722.